extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -284,26 +278,6 @@ export class AmpPanZoom extends AMP.BaseElement {
       : defaultValue;
   }
 
-  /**
-   * @return {number}
-   * @private
-   */
-  getYOffsetFromCenter_() {
-    const elementContentGap = this.elementBox_.height - this.contentBox_.height;
-    const distanceToCenter = elementContentGap / 2;
-    return this.contentBox_.top - distanceToCenter;
-  }
-
-  /**
-   * @return {number}
-   * @private
-   */
-  getXOffsetFromCenter_() {
-    const elementContentGap = this.elementBox_.width - this.contentBox_.width;
-    const distanceToCenter = elementContentGap / 2;
-    return this.contentBox_.left - distanceToCenter;
-  }
-
   /**
```

Comment:
It would be better to provide a more illustrative way of describing what are we actually measuring here, as a bunch of ratio calculations are involved here and does not seem very clear at first sight

Reply:
This is removed. I guess you want the ASCII art above `updatePanZoomBounds_`?

Reply:
@cathyxz I am referring to `measure_`. It seems that Github is somehow placing the comments in a confusing way...

Reply:
Oh gotcha. Sure thing.

Reply:
Refactored this function and added some comments.